backend/repositories/stores/menus/index.go

```
@@ -26,15 +30,39 @@ func (r *MenuRepository) GetMenusByStoreID(c context.Context, storeID uuid.UUID)
 	return db2entities.ToMenus(dbMenus), err
 }
 
-func (r *MenuRepository) CreateMenu(c context.Context, input menuService.CreateMenuRepositoryInput) (menus.Menu, error) {
-	m, err := r.queries.CreateMenu(c, sqlc.CreateMenuParams{
+func (r *MenuRepository) CreateMenuWithToppings(c context.Context, input menuService.CreateMenuRepositoryInput) (menus.Menu, error) {
+	tx, err := r.pool.Begin(c)
+	if err != nil {
+		return menus.Menu{}, err
+	}
+	defer tx.Rollback(c)
```

golangci-lint (2.12.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)